fr/canada-2018.md

```diff
@@ -0,0 +1,81 @@
+# NomadConf Canada 2018
+
+Cette édition aura lieu au Canada dans la première quinzaine de novembre 2018 et est organisée par Clémentine Hahn et Matti Schneider.
+
+_Toutes les informations indiquées ici sont en cours de confirmation, nous sommes preneurs de tous vos retours !_
+
+
+## Thème : quand ralentir nous permet d'aller plus vite, plus loin… ou simplement ailleurs ?
+
+Pour cette édition, nous souhaitons explorer le rapport au temps et à la productivité, et notamment la notion selon laquelle prendre un temps plus long pour mener des idées à terme peut permettre d'atteindre un résultat plus rapidement qu'un démarrage immédiat de la production.
+Sans revenir au cycle en V, il s'agit de questionner les pratiques agiles et lean pour identifier les occasions où nous présageons à tort que le fait d'aller vite aurait une vertu intrinsèque et auto-suffisante. Il s'agit des cas où nous pouvons, en tant que praticien·ne·s, confondre des applications _tactiques_ de l'itération rapide et de la confrontation à la réalité avec des applications _stratégiques_ qui empêchent de prendre du recul et d'assurer l'adéquation entre l'intention initiale et la mise en œuvre de solutions. Bref, nous chercherons à déterminer les moments où nous mélangeons fin et moyens.
```

Comment:
Je pense que je comprend l'idée parce-que je la connaissais avant, mais après deux relectures, je continue à ne pas comprendre la phrase : 
> confondre des applications _tactiques_ de l'itération rapide et de la confrontation à la réalité avec des applications _stratégiques_ qui empêchent de prendre du recul

Autre question : qu'apporte le ", en tant que praticien·ne·s," ?

Reply:
@MattiSG :  Notamment je ne vois pas en quoi une "applications _stratégiques_" empêche de "prendre du recul".

Je ne vois pas ce que tu veux dire par "applications _tactiques_" et "applications _stratégiques_" de l'itération rapide. Je me raccroche à l'idée de tactique et de stratégie, mais ça me paraît en contradiction avec ce que tu écrits.

Reply:
@MattiSG mon intuition naturelle serait de ne pas partir sur une affirmation pour le titre "Quand ralentir permet de..."  Mais plutôt de poser une question qui contiendrait la posture contre-intuitive qu'on a envie d'aborder.  Qu'est-ce que l'efficacité (je n'aime pas ce mot mais je n'en trouve pas d'autres là tout de suite)  sinon résoudre le problème que l'on s'est fixé ? Et pour résoudre un problème quelle est la mesure ? Le temps passé à la résoudre ou le temps de pérennité de la résolution ? Finir de résoudre un problème vite ne veut pas dire que c'est un succès. Résoudre un problème dans le temps long, ça c'est un succès. Mais il est difficile de penser le temps long ajd. 

J'ai la sensation que parler de fin et moyens est plus explicite que de tactique et stratégie. Peut-être on pourrait essayer d'expliciter un poil plus ? ci-après quelques mots qui me sont venus, ils peuvent tout à fait être virés ou amendés c'est ce qui m'est venu en tête après lecture de ton texte.  "la méthode agile recommande de tester vite, de se confronter à l'utilisateur vite Mais pourquoi ? Pas pour produire vite dans l'absolu. Plutôt pour avoir des éléments de réflexions concrets vite. Pour nourrir une pensée qui elle peut/doit ? avoir lieu au long cours. Aller vite n'est pas un but en soi. Pour résoudre un problème de façon efficace il faut avoir conscience du temps long, alors prenons le ensemble."  
Je ne sais pas s'il faut ou pas  intégrer ça dans ta proposition ni comment. Parlons-en :)